api/src/test/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminServiceTest.java

```diff
@@ -315,7 +312,7 @@ public void testGetWorkspaceAdminView() {
 
   @Test
   public void testGetWorkspaceAdminView_published_false() {
-    workspaceDao.save(dbWorkspace.setPublished(false));
+    workspaceDao.save(dbWorkspace.setPublished(false).setFeaturedCategory(null));
```

Comment:
Is this written this way because we have to accommodate both the new and old ways of publishing?

Reply:
Yes, until we clean up the old code, publish column in dbworkspace, doesnt have any impact on featured workspace category